### ISSUE TITLE
Asterisk: New release 18.6.0

### ISSUE THREAD

**Issue by jospezial** [openwrt/telephony#678]
Maintainer: @jslachta
Description:
PKG_VERSION:=18.6.0
PKG_HASH:=1a13858dfb68bfbc3236ef8e30afaf86320f8c7e94c0e1a5135e7c25d0f6ab56

090-app_dial-expanded-A-option-to-add-caller-announcement.patch has to be deleted because the changes are already in this release.

Still compiling my image. Will report bugs of the run test if I find any.

**Comment by jospezial** [comment]
No regressions so far in my run test.
My lantiq dtmf bug remains.
@micmac1 do you like to create a pr for the new release?

**Comment by micmac1** [comment]
No. I don't see the urgency. The two security bugs mentioned in the
release notes seem to be addressed already in 18.5.1. Correct me if I'm
wrong.
